test/postgresql.migration.test.js

```diff
@@ -109,6 +109,32 @@ describe('migrations', function() {
       done();
     });
   });
+
+  it('should add default value for new required columns', async function() {
+    const DefaultValueAfterColumnAdd = await db.getModel('DefaultValueAfterColumnAdd');
+    await DefaultValueAfterColumnAdd.create({
+      name: 'name1',
+    });
+    await db.defineProperty('DefaultValueAfterColumnAdd', 'createdByAdmin', {
```

Comment:
`defineProperty` is a sync function I think, see [its definition](https://github.com/strongloop/loopback-datasource-juggler/blob/master/lib/datasource.js#L1017)